front/lib/auth.ts

```diff
@@ -671,9 +691,11 @@ export class Authenticator {
   }
 
   hasPermission(acls: ACLType[], permission: Permission): boolean {
-    // Does the user belongs to a group which has the required permission on all ACLs ?
-    return this.groups().some((group) =>
-      acls.every((acl) => groupHasPermission(acl, permission, group.id))
+    // For each acl, does the user belongs to a group that has the permission?
```

Comment:
:warning: Raising attention here, reviewers
Pretty sure of this change (`every` has `some` instead of `some` has every; but quite critical to get right so if one of you can double check

Reply:
We want indeed want `\forall a \in ACL, \exists g \in Groups(user): groupHasPermission(acl, g)` 

And we indeed had `\exist g, \forall a` which seems wrong I agree.

Reply:
cc @tdraier @flvndvd

Reply:
Aligned 👍